python/setup.py

```diff
@@ -51,11 +51,13 @@
 # Various parameters depend on whether we are the lite package or not
 if lite:
     pkgname = 'phonenumberslite'
-    pkgs = ['phonenumbers', 'phonenumbers.data', 'phonenumbers.shortdata']
+    pkgs = ['phonenumbers', 'phonenumbers.data', 'phonenumbers.shortdata',
+            'tests', 'tests.testdata']
```

Comment:
Could you explain this change (and `MANIFEST.in`) for me?  (My knowledge/memory of Python packaging is poor/rusty.)

Reply:
This isn't explicitly required, but makes it such that only the relevant test files get included in each package -- which should also reduce publication sizes on pypi, etc.

I see `MANIFEST.in` files as a bit of black magic and try to make them as simple as possible, so putting declarative config into setup helps that!